gravityflow/gravityflowwoocommerce
# Fix an issue with Gravity Forms 2.5 not displaying the Field Mappings of WooCommerce

## Description
Re: [Issue#39](https://github.com/gravityflow/backlog/issues/39)
Field mappings not showing up in WooCommerce feed settings with Gravity Forms 2.5

## Testing instructions
1. Install the current beta of Gravity Forms 2.5 and enable the 'fix-gravityforms-2.5' branch for Gravity Flow.
2. Check feed settings for Gravity Flow WooCommerce (master) on a form,  the feed settings should not be appearing.
3. Switch to this branch and check if the feed settings appear now.
4. Revert from Gravity Forms 2.5 beta to Gravity Forms 2.4 to ensure nothing breaks on the Gravity Flow WooCommerce feed settings.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
